lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -1395,6 +1397,25 @@ void ModuleEmitter::emitStatement(rtl::ConnectOp op) {
   emitLocationInfoAndNewLine(ops);
 }
 
+void ModuleEmitter::emitStatement(rtl::OutputOp op) {
```

Comment:
Plz add a comment saying something like "for output op we emit the assign statements at the end of the module that set all the output wires"

Reply:
Comment in the C++ code or the generated SV code?

Reply:
I mean in the c++ code.